stake-pool/js/src/index.ts

```diff
@@ -46,6 +46,7 @@ import BN from 'bn.js';
 export type { StakePool, AccountType, ValidatorList, ValidatorStakeInfo } from './layouts';
 export { STAKE_POOL_PROGRAM_ID } from './constants';
 export * from './instructions';
+export { StakePoolLayout } from './layouts';
```

Comment:
If we're going to make this one public, might as well do `ValidatorStakeInfoLayout` and `ValidatorListLayout` as well.